pegasus/helper_modules/dashboard.rb

```diff
@@ -0,0 +1,93 @@
+#
+# A set of utilities that provide access to the current user, their
+# authentication state, and other DASHBOARD_DB access.
+#
+module Dashboard
+
+  # Get the dashboard database
+  # The static constant is wrapped here so it's easy to stub for tests.
+  # @returns [Database]
+  def self.db
+    DASHBOARD_DB
+  end
+
+
+  def self.admin?(user_id)
+    !!db[:users].where(id: user_id, admin: true).first
+  end
+
+  class User
+    # Wrap dashboard user row in this helper object.
+    # You can use this, but it's preferred that clients call User.get(user_id).
+    def initialize(user_row)
+      @row = user_row
+    end
+
+    # @returns [User] for given user_id, or nil if not found in database
+    def self.get(user_id)
+      return nil unless (row = Dashboard::db[:users][id: user_id])
```

Comment:
Avoid assignment side-effects in if/unless conditions.  (Tricky and too easy to mix up = vs. ==)
How about: